src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/erasure-code-profile-form/erasure-code-profile-form.component.ts

```diff
@@ -0,0 +1,252 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { Validators } from '@angular/forms';
+
+import { BsModalRef } from 'ngx-bootstrap/modal';
+
+import { ErasureCodeProfileService } from '../../../shared/api/erasure-code-profile.service';
+import { CdFormBuilder } from '../../../shared/forms/cd-form-builder';
+import { CdFormGroup } from '../../../shared/forms/cd-form-group';
+import { CdValidators } from '../../../shared/forms/cd-validators';
+import { ErasureCodeProfile } from '../../../shared/models/erasure-code-profile';
+import { FinishedTask } from '../../../shared/models/finished-task';
+import { TaskWrapperService } from '../../../shared/services/task-wrapper.service';
+import { ErasureCodeProfileFormTooltips } from './erasure-code-profile-form-tooltips';
+
+@Component({
+  selector: 'cd-erasure-code-profile-form',
+  templateUrl: './erasure-code-profile-form.component.html',
+  styleUrls: ['./erasure-code-profile-form.component.scss']
+})
+export class ErasureCodeProfileFormComponent implements OnInit {
+  @Output()
+  submitAction = new EventEmitter();
+
+  form: CdFormGroup;
+  failureDomains: string[];
+  plugins: string[];
+  names: string[];
+  techniques: string[];
+  requiredControls: string[] = [];
+  devices: string[] = [];
+  tooltips = new ErasureCodeProfileFormTooltips();
+
+  PLUGIN = {
+    LRC: 'lrc', // Locally Repairable Erasure Code
```

Comment:
Why not LREC? Otherwise SHC would be sufficient. Or is this name used by Ceph in general?

Reply:
The name is chosen by Ceph or the authors of those plugins. It seems to me too, that they have no name convention between the plugins.